src/lib.rs

```diff
@@ -143,6 +148,18 @@ fn makecircle(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Poly
     Polygon::new(LineString::from(circle_points), vec![])
 }
 
+// Make circle using geographic (lon/lat) coordinates
+fn makecircle_geo(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Polygon<f64> {
+    let mut circle_points = Vec::new();
+    let g = Geodesic::wgs84();
+    for i in 0..num_vertices {
+        let angle: f64 = 360.0 / (num_vertices as f64) * (i as f64);
+        let (x, y, az) = g.direct(centerpoint.x(), centerpoint.x(), angle, radius * 50000.0);
```

Comment:
```suggestion
        let (x, y, az) = g.direct(centerpoint.x(), centerpoint.y(), angle, radius * 50000.0);
```
typo?

Reply:
What's the expected units of input `radius`? 

GeographicLib expects meters, so I'm surprised to see the * 50,000 - but maybe it's just for some testing purpose. I realize this is still "draft" code, so apologies for jumping to any conclusions...

Reply:
Correct, just for testing. Previously I ignored units but degrees are pretty close to 100 km it turns out so that is what generated the exclamation mark.